R/deployDoc.R

```diff
@@ -58,9 +58,14 @@ standardizeSingleDocDeployment <- function(path,
 
     appFiles <- c(basename(path), resources$path)
 
-    if (file.exists(file.path(dirname(path), ".Rprofile"))) {
-      appFiles <- c(appFiles, ".Rprofile")
+    if (isRenderedFile(path)) {
+      for (each in c(".Rprofile", "renv.lock", "requirements.txt")) {
+        if (file.exists(file.path(dirname(path), each))) {
+          appFiles <- c(appFiles, each)
+        }
+      }
```

Comment:
```suggestion
      candidates <- c(".Rprofile", "renv.lock", "requirements.txt")
      exists <- file.exists(file.path(dirname(path), candidates))
      appFiles <- c(appFiles, candidates[exists])
```

Not sure if I got all the parens in the right place, but this is a vectorisation win 😄

Reply:
Thanks. It wasn't obvious to me that `file.path` did vectors!